### ISSUE TITLE
Unexpected behavior when testing with Jest—disappearing instance methods

### ISSUE THREAD

**Issue by neezer** [zeromq/zeromq.js#416]
**Describe the bug**
Method `send` on instance of `zmq.Publisher` vanishes on subsequent test runs in Jest with `--watch --detectOpenHandles --runInBand`. The test passes on the first run, but subsequent runs fail with

```
● works always

    expect(received).toBe(expected) // Object.is equality

    Expected: function
    Received: undefined
```

- The test passes on subsequent runs if both `--runInBand` and `--detectOpenHandles` flags are omitted.
- Test fails on subsequent runs if either/both `--runInBand` or/and `--detectOpenHandles` flags is/are provided.

**Reproducing**
Using `jest@25.2.7`, define the following test:

```js
const zmq = require("zeromq");
const addr = "inproc://some-addr";

let publisher;

afterEach(async () => {
  if (publisher) await publisher.unbind(addr);
});

test("works always", async () => {
  publisher = new zmq.Publisher();

  await publisher.bind(addr);

  expect(typeof publisher.send).toBe("function");
});
```

1. Run with `jest path/to/above/test/file.js --watch` with either/both `--detectOpenHandles` or/and `--runInBand`.
1. Trigger multiple test runs without closing the test runner instance.

    Eg., hit the <kbd>Enter</kbd> key with the test runner focused

**Expected behavior**
This test should always pass.

**Tested on**
 - OS: macOS 10.15.6
 - ZeroMQ.js version: 6.0.0-beta.6

**Comment by neezer** [comment]
Exhibits the same behavior if you skip the binding:

```js
const zmq = require("zeromq");

let publisher;

test("works always", async () => {
  publisher = new zmq.Publisher();

  expect(typeof publisher.send).toBe("function");
});
```

Also if you define a new `publisher` on each run:

```js
const zmq = require("zeromq");

test("works always", async () => {
  const publisher = new zmq.Publisher();

  expect(typeof publisher.send).toBe("function");
});
```

**Comment by ishantiw** [comment]
@neezer do we have any update on this issue yet?

**Comment by neezer** [comment]
@ishantiw No, sorry. I encountered this issue on a side project and haven't had time to return to it.

**Comment by ptitjes** [comment]
This already merged commit https://github.com/zeromq/zeromq.js/commit/4b3aad8faaafc4790cba18507f7194850e3f4d84 seems to fix the problem.

It would be great to have a 6.0.0-beta.7 or better a final version. The last 6.0.0 version is from the 17 Dec 2019!
